OpenSim/Simulation/Wrap/WrapObjectSet.h

```diff
@@ -28,6 +28,8 @@
 
 namespace OpenSim {
 
+/** @cond **/ // hide from Doxygen
```

Comment:
I would expose this class since this Set appears in files and is rather thin class.